lvsfunc/deblock.py

```diff
@@ -151,7 +155,7 @@ def dpir(
     matrix: Matrix | int | None = None, cuda: bool | Literal['trt'] | None = None, i444: bool = False,
     tiles: int | Tuple[int, int] | None = None, overlap: int | Tuple[int, int] | None = 8,
     zones: List[Tuple[Range | List[Range] | None, SupportsFloat | vs.VideoNode | None]] | None = None,
-    fp16: bool | None = None, num_streams: int = 1, device_id: int = 0, kernel: Kernel | str = Catrom()
+    fp16: bool | None = None, num_streams: int = 1, device_id: int = 0, kernel: Kernel | str = Bicubic(b=0, c=0.5)
```

Comment:
Why? Those are even the default values for Bicubic itself

Reply:
Clarity.

Reply:
I’d rather have `Catrom`.